Scripts/Core/VRView.cs

```diff
@@ -132,9 +133,8 @@ public void OnEnable()
 			m_Camera.farClipPlane = 1000f;
 
 			// Generally, we want to be at a standing height, so default to that
-			const float kHeadHeight = 1.7f;
 			Vector3 position = m_CameraRig.position;
-			position.y = kHeadHeight;
+			position.y = HeadHeight;
```

Comment:
Is this information provided from the partner SDKs and/or Unity API? Would be good to have the default HeadHeight at 1.7 unless a custom value is provided from the SDK.